apps/editor.planx.uk/src/@planx/components/FileUploadAndLabel/Editor.tsx

```diff
@@ -191,32 +193,30 @@ function FileTypeEditor(props: ListManagerEditorProps<FileType>) {
       </InputRow>
       {isConditionalRule && (
         <InputRow>
-          <Input
+          <DataFieldAutocomplete
             required
-            name="fn"
             value={props.value.rule.fn}
-            onChange={(e) =>
+            onChange={(value) =>
               props.onChange(
                 merge(props.value, {
                   rule: {
-                    fn: e.target.value,
+                    fn: value,
                   },
                 }),
               )
             }
-            placeholder="Data field"
             disabled={props.disabled}
           />
           <Operator>Equals</Operator>
-          <Input
+          <DataFieldAutocomplete
             required
-            name="val"
+            schema={["recommended", "required"]}
```

Comment:
@jessicamcinchak Really appreciate this bit of design 👍 Very nice to be able to suggest any values, but not constrain the user!